pkg/sql/schemachanger/scplan/internal/rules/dep_drop_index.go

```diff
@@ -105,3 +105,27 @@ func init() {
 	)
 
 }
+
+func init() {
+	registerDepRuleForDrop(
+		"view can only go to absent when the index becomes hidden",
```

Comment:
Can you reformulate this rule name in a similar style as the others? Otherwise when we have to look at the corpus of rules it can become very confusing very quickly.

The `from` element is usually referred to before `to`, as in "'from' is in whatever state (before/right before) 'to' is in whatever state".